bridges/AtmoNouvelleAquitaineBridge.php

```diff
@@ -0,0 +1,134 @@
+<?php
+class AtmoNouvelleAquitaineBridge extends BridgeAbstract {
+
+	const NAME = 'Atmo Nouvelle Aquitaine';
+	const URI = 'https://www.atmo-nouvelleaquitaine.org/monair/commune/33063';
+	const DESCRIPTION = 'Fetches the latest air polution of Bordeaux from Atmo Nouvelle Aquitaine';
+	const MAINTAINER = 'floviolleau';
+	const PARAMETERS = array();
+	const CACHE_TIMEOUT = 7200;
+
+	private $dom;
+
+	private function getClosest($search, $arr) {
+		$closest = null;
+		foreach ($arr as $key => $value) {
+			if ($closest === null || abs((int)$search - $closest) > abs((int)$key - (int)$search)) {
+				$closest = (int)$key;
+			}
+		}
+		return $arr[$closest];
+	}
+
+	public function collectData() {
+		$uri = self::URI;
+
+		$html = getSimpleHTMLDOM($uri)
+				or returnServerError('Could not request ' . $uri);
+
+		$this->dom = $html->find('#block-system-main .city-prevision-map', 0);
+
+		$message = $this->getIndexMessage() . ' ' . $this->getQualityMessage();
+		$message .= ' ' . $this->getTomorrowTrendIndexMessage() . ' ' . $this->getTomorrowTrendQualityMessage();
+
+		$item['uri'] = $uri;
+		$today = date('d/m/Y');
+		$item['title'] = "Bulletin de l'air du $today pour la région Nouvelle Aquitaine.";
+		$item['title'] .= " Retrouvez plus d'informations en allant sur atmo-nouvelleaquitaine.org #QualiteAir.";
+		$item['author'] = 'floviolleau';
```

Comment:
The author here refers to the data present in the feed. Here it should probably be `atmo-nouvelleaquitaine` or something.

Reply:
Commonly, I prefer to put double quotes if the string contains a variable(s) and simple quotes if the string doesn't contains a variable(s) so that it allows php to be able to fastly interpret the simple quote string (php will know that this string doesn't contains a variable(s) to handle)

Reply:
Yes the data are on the website atmo-nouvelleaquitaine.org. It's for air quality measures